js/apps/admin-ui/package.json

```diff
@@ -126,16 +117,15 @@
     "@types/react": "^18.3.18",
     "@types/react-dom": "^18.3.5",
     "@vitejs/plugin-react-swc": "^3.8.0",
-    "cypress": "^14.1.0",
-    "cypress-axe": "^1.6.0",
-    "cypress-split": "^1.24.14",
     "jsdom": "^26.0.0",
     "ldap-server-mock": "^6.0.1",
     "lightningcss": "^1.29.1",
     "ts-node": "^10.9.2",
+    "uuid": "^11.0.3",
     "vite": "^6.2.0",
     "vite-plugin-checker": "^0.9.0",
     "vite-plugin-dts": "^4.5.1",
-    "vitest": "^3.0.8"
+    "vitest": "^3.0.8",
+    "uuid": "^11.0.3"
```

Comment:
This dependency is listed twice, likely because of a merge conflict being resolved incorrectly.

Reply:
![image](https://github.com/user-attachments/assets/cd27c80e-06a8-4b25-bc22-37ae24ec93c0)

Reply:
ahh, that must have been a merge that added that double some how, fixed that

Reply:
Nice, this seems fixed now.